src/assets/subjects/subjects.json

```diff
@@ -0,0 +1,206 @@
+{
```

Comment:
This file shouldn't be in this PR right?

Reply:
I didn't intend it at the start, but I branched off of the group subjects by colors branch and just went with it after a few commits. If we want to remove the colored schema for subjects, what if we merge this branch into the group subjects by colors and then remove the features?